generators/client/templates/src/main/webapp/app/modules/administration/audits/audits.js

```diff
@@ -2,12 +2,15 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import Translate from 'react-translate-component';
 
+import { Table, TableBody, TableHeader, TableHeaderColumn, TableRow, TableRowColumn } from 'material-ui/Table';
+
 import { getAudits } from '../../reducers/administration';
 
 export class AuditsPage extends Component {
 
   constructor(props) {
     super(props);
+    this.state = { showCheckboxes: false };
```

Comment:
I dont think this needs to be the component state.. we can set false on the declarations on the component declare it as  const in render method and use that. Ill merge that do that